cmd/list.go

```diff
@@ -82,102 +80,93 @@ func (lr *listResult) print() {
 }
 
 func showComponentList(env *meta.Environment, onlyInstalled bool, refresh bool) (*listResult, error) {
-	if refresh || env.Profile().Manifest() == nil {
-		manifest, err := env.Repository().Manifest()
-		if err != nil {
-			return nil, err
-		}
-		err = env.Profile().SaveManifest(manifest)
-		if err != nil {
-			return nil, err
-		}
+	index, err := env.V1Repository().FetchIndexManifest()
+	if err != nil {
+		return nil, err
 	}
 
 	installed, err := env.Profile().InstalledComponents()
 	if err != nil {
 		return nil, err
 	}
-	manifest := env.Profile().Manifest()
+
 	var cmpTable [][]string
-	cmpTable = append(cmpTable, []string{"Name", "Installed", "Platforms", "Description"})
+	cmpTable = append(cmpTable, []string{"Name", "Owner", "Installed", "Platforms", "Description"})
```

Comment:
I think it would be good for `tiup list` to give a very simple list of component names and `tiup list --verbose` to give this view. It is good info, but I think it might be overwhelming for new users.

Reply:
This also has the advantage that `tiup list` would be very fast because it would only need the index manifest and installed versions.

Reply:
what do you think about this? @lonng

Reply:
@nrc We will have `verbose` in the next minor version